torch/quantization/custom_module_class_mappings.py

```diff
@@ -9,15 +11,21 @@ def register_observed_custom_module_mapping(float_custom_module_class, observed_
     This will be used in prepare step of post training static quantization or
     quantization aware training
     """
+    global OBSERVED_CUSTOM_MODULE_CLASS_MAPPINGS
     assert hasattr(observed_custom_module_class, 'from_float'), 'from_float must be' + \
         ' defined in observed custom module class'
     OBSERVED_CUSTOM_MODULE_CLASS_MAPPINGS[float_custom_module_class] = \
         observed_custom_module_class
 
-def get_observed_custom_module_class(float_custom_module_class):
+def get_observed_custom_module_class(float_custom_module_class=None):
     """ Get the corresponding observed module class for a given
     float custom module.
     """
+    global OBSERVED_CUSTOM_MODULE_CLASS_MAPPINGS
+    if float_custom_module_class is None:
+        return OBSERVED_CUSTOM_MODULE_CLASS_MAPPINGS
```

Comment:
what's the context for adding this?  Seems unexpected, since this is changing the return type from a class to a mapping of classes.

Reply:
The point on the return type makes sense -- the motivation was for debugging purposes, sometimes I needed to see what was registered so far. There was no function for it, so I piggybacked on that function. I will revert it back